sbin/build.sh

```diff
@@ -1125,7 +1125,7 @@ addVendorToJson(){
 }
 
 addSourceToJson(){ # Pulls the basename of the origin repo, or uses 'openjdk-build' in rare cases of failure
-  local repoName=$(basename -s .git $(cd ${BUILD_CONFIG[WORKSPACE_DIR]} && git config --get remote.origin.url 2>/dev/null))
+  local repoName=$(basename $(cd ${BUILD_CONFIG[WORKSPACE_DIR]%.git} && git config --get remote.origin.url 2>/dev/null))
```

Comment:
Should this not be contained in quotes to avoid the % splitting out?
```suggestion
  local repoName=$(basename $(cd "${BUILD_CONFIG[WORKSPACE_DIR]%.git}" && git config --get remote.origin.url 2>/dev/null))
```

Reply:
I don't think there's any situation in which that specifically would be a problem given that it's within the `${ }` (interested to see if you can make one up though!) .. And as far as I can see even our new aggressive linting hasn't objected to this, but I can change it